certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -62,6 +62,15 @@
     ['#', " } # managed by Certbot"],
 ]
 
+PREVIOUS_SSL_OPTIONS_HASHES = [
+    '0f81093a1465e3d4eaa8b0c14e77b2a2e93568b0fc1351c2b87893a95f0de87c',
+    '9a7b32c49001fed4cff8ad24353329472a50e86ade1ef9b2b9e43566a619612e',
+    'a6d9f1c7d6b36749b52ba061fff1421f9a0a3d2cfdafbd63c05d06f65b990937',
+    '7f95624dd95cf5afc708b9f967ee83a24b8025dc7c8d9df2b556bbc64256b3ff',
+]
+
+CURRENT_SSL_OPTIONS_HASH = '394732f2bbe3e5e637c3fb5c6e980a1f1b90b01e2e8d6b7cff41dde16e2a756d'
```

Comment:
nit: I'm sorry I didn't bring this up before, but it may be worth moving these values into `constants.py` given that's the purpose of the file and it's a lot smaller that configurator.py. That file also has docstrings for all of the constants which I think is nice.

You can ignore this comment if you want though.